### PR TITLE
SDK-Proto: GET item, item-type to return all relevant fields

### PR DESCRIPTION
**Pull request by rudyrusli**
Ticket : https://mythicalgames.atlassian.net/browse/MYTHOS-740

### CODE REVIEW

File: api/itemtype/definition.proto [hunk 1]
```diff
@@ -13,23 +12,19 @@ package saga.api.itemtype;
...
-    /* The game's unique id for this ItemType */
-    string game_item_type_id = 2;
-    /* Name of this ItemType */
-    string name = 3;
-    /* Title this ItemType is associated with */
-    string title_id = 5;
-    PriRevShareSettings pri_rev_share_settings = 6;
-    SecRevShareSettings sec_rev_share_settings = 7;
-    /* Is this item withdrawable? */
-    bool withdrawable = 8;
```
Comment (axcrest) [inline api/itemtype/definition.proto:26]: I think we should keep withdrawable

**Review by axcrest — APPROVED** [review]

File: api/itemtype/definition.proto (hunk 1 above)
Comment (rudyrusli) [reply api/itemtype/definition.proto:26]: Sounds good. I have restored them now. 
Do we have to worry where we get it for now (ItemService doesn't return it at the moment) ?

Comment (axcrest) [reply api/itemtype/definition.proto:26]: it can just be false for now since its not actually used/implemented yet

**Review by axcrest — APPROVED** [review]

File: api/itemtype/definition.proto (hunk 1 above)
Comment (rudyrusli) [reply api/itemtype/definition.proto:26]: Ok, gotcha.

**Pull request merged**